src/pages/test-report/components/competency/competency.ts

```diff
@@ -6,22 +6,28 @@ import { merge } from 'rxjs/observable/merge';
 import { map } from 'rxjs/operators';
 
 import { StoreModel } from '../../../../shared/models/store.model';
-import { AddDrivingFault } from '../../../../modules/tests/test_data/test-data.actions';
+import { AddDrivingFault, AddSeriousFault } from '../../../../modules/tests/test_data/test-data.actions';
 import { HammerProvider } from '../../../../providers/hammer/hammer';
 import { Competencies } from '../../../../modules/tests/test_data/test-data.constants';
 import { competencyLabels } from './competency.constants';
 import { getCurrentTest } from '../../../../modules/tests/tests.selector';
 import { getTestData } from '../../../../modules/tests/test_data/test-data.reducer';
-import { getDrivingFaultCount } from '../../../../modules/tests/test_data/test-data.selector';
 import { getTests } from '../../../../modules/tests/tests.reducer';
+import { getDrivingFaultCount, hasSeriousFault } from '../../../../modules/tests/test_data/test-data.selector';
+import { getTestReportState } from '../../test-report.reducer';
+import { isSeriousMode } from '../../test-report.selector';
+import { ToggleSeriousFaultMode } from '../../test-report.actions';
 
 enum CssClassesEnum {
   DRIVING_FAULT = 'driving-fault',
   RIPPLE_EFFECT = 'ripple-effect',
+  SERIOUS_FAULT = 'serious-fault',
 }
 
 interface ComptencyState {
```

Comment:
okay - this is an actual typo of an interface name. that would bug me forever 

ComptencyState should be CompetencyState